docs/content/doc/developers/guidelines-frontend.md

```diff
@@ -40,25 +40,30 @@ We recommend [Google HTML/CSS Style Guide](https://google.github.io/styleguide/h
 1. Every feature (Fomantic-UI/jQuery module) should be put in separate files/directories.
 2. HTML ids and classes should use kebab-case.
 3. HTML ids and classes used in JavaScript should be unique for the whole project, and should contain 2-3 feature related keywords. We recommend to use the `js-` prefix for classes that are only used in JavaScript.
-4. jQuery events across different features should use their own namespaces.
-5. CSS styling for classes provided by frameworks should not be overwritten. Always use new class-names to overwrite framework styles. We recommend to use the `us-` prefix for user defined styles.  
+4. jQuery events across different features could use their own namespaces if there are potential conflicts.
```

Comment:
I would revert this. Switching `should` to `could` puts the burden on developers to have all other occurences in mind.

I can see the potential of consolidating, though. But Loïc Dachary made the case on the chat that tracing down the source of something is important for maintenance.

Reply:
The same to the `us-` prefix, it's hard to be followed (and in most cases there is no problem). To reduce pressures, `could` sounds more friendly ....